src/hooks/visits.ts

```diff
@@ -46,7 +46,7 @@ export const useMutateVisits = () => {
   const queryClient = useQueryClient()
   const { setAlert } = useAlert()
 
-  const mutationFn = async (visit: Visit & { deleteDoc?: boolean }) => {
+  const mutationFn = async (visit: Visit | { docId?: string }) => {
```

Comment:
Nice change, is it worth doing this for some of the other hooks?

Reply:
User mutation hook is a little funky now but I don't want to refactor